arping.c

```diff
@@ -844,6 +844,8 @@ static int event_loop(struct run_state *ctl)
 	close(sfd);
 	close(tfd);
 	freeifaddrs(ctl->ifa0);
+	if (err < 0)
+                return err;
```

Comment:
Unlike ping, which exits on error with `1` or `2` (depending whether exit code is related by ping error or other error), arping specifies just `1`. IMHO the exit code should be also `2`.

`-1` will be translated to `255` due `status & 0xFF` (see [`man exit(3)`](https://man7.org/linux/man-pages/man3/exit.3.html).

Also, we would need to print error message, use [`strerror(3)`](https://man7.org/linux/man-pages/man3/strerror.3.html).

Reply:
The 'err' here is the return value of sendto(). According to https://man7.org/linux/man-pages/man3/sendto.3p.html, the return value of sendto() returned from send_pack() when failed should be -1, shall we consider other scenarios of failure?I add a print of error message using strerror, should we just do it in send_pack() or out?